Source/Private/ASElementMap.h

```diff
@@ -55,6 +60,11 @@ AS_SUBCLASSING_RESTRICTED
 - (nullable NSIndexPath *)indexPathForElement:(ASCollectionElement *)element;
 
 /**
+ * Returns the index path for the given element, if it represents an item. O(1)
+ */
+- (nullable NSIndexPath *)indexPathForElementIfItem:(ASCollectionElement *)element;
```

Comment:
Good call. I went with `indexPathForElementIfCell:` which I think is a lot clearer. I try to avoid using the word `Cell` in general but there is precedent in `UICollectionElementCategoryCell`.